target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -401,6 +401,17 @@ define Device/tplink_eap225-outdoor-v1
 endef
 TARGET_DEVICES += tplink_eap225-outdoor-v1
 
+define Device/tplink_eap225-outdoor-v3
+  $(Device/tplink-eap2x5)
+  SOC := qca9563
+  IMAGE_SIZE := 13824k
```

Comment:
This is the image size for the partition layout from eap2x5-1port.dtsi. Here you overwrite the partition layout, so you *could* `IMAGE_SIZE := 15104k`. That's 1280kB, but shouldn't be an issue for now. I think we should either keep the original layout, or change it for all devices (this was discussed previously, but I would have to check some old PRs and the mailing list).

Reply:
Unless there is some showstopping reason, as a general policy IMAGE_SIZE, DTS "firmware" and tplink-safeloader "firmware" should always be the same size and only overwrite `os-image` and `file-system`.

When they are different it just causes too much of a mess, is confusing, unnecessarily introduces edge cases, no-one can remember why it was done like that, it breaks sysupgrade-to-OEM (even if TFTP is still the main way to restore), etc. People end up just blindly copy-pasting the pattern because its too confusing to trace and its whats the older devices do, instead of matching the current "partition-table" in the OEM firmware (E.g. the CPE devices).

Much simpler and safer just to keep them all the same as intended, which thankfully seems to be the case for this patch:) Just commenting for future reference!